docker/iml-api.dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM rust-iml-base as builder
+
+FROM ubuntu
```

Comment:
I haven't looked at the size difference but may want to consider using centos:7

Reply:
yes, waiting on your changes :)